assets/queries/cloudFormation/elastic_search_without_slow_logs/query.rego

```diff
@@ -0,0 +1,54 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].Resources[name]
+	resource.Type == "AWS::Elasticsearch::Domain"
+	logs := resource.Properties.LogPublishingOptions[log]
+	not contains(["INDEX_SLOW_LOGS", "SEARCH_SLOW_LOGS"], log)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("Resources.%s.Properties.LogPublishingOptions.%s", [name, log]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("Resources.%s.Properties.LogPublishingOptions.%s is slow logs", [name, log]),
+		"keyActualValue": sprintf("Resources.%s.Properties.LogPublishingOptions.%s is not not slow logs", [name, log]),
+	}
+}
```

Comment:
Should this query trigger every time a Log is not a `Slow_Log`, or when none of the Logs of the Domain is a `Slow_Log`?

Reply:
In case of Terraform and Ansible it should tigger because the variable is not and array of logs, in this case it is, so i'm not sure...